packages/design-system/src/components/OcRecipient/OcRecipient.spec.ts

```diff
@@ -83,16 +77,14 @@ describe('OcRecipient', () => {
       }
     ],
     ['icon name is not defined', { name: 'Alice', icon: {} }],
-    ['icon name is not a string', { name: 'Alice', icon: { name: { inverted: 'inverted' } } }],
+    ['icon name is not a string', { name: 'Alice', icon: { name: null } }],
     ['icon name is empty', { name: 'Alice', icon: { name: '' } }],
     ['icon label is not defined', { name: 'Alice', icon: { name: 'person' } }],
-    [
-      'icon label is not a string',
-      { name: 'Alice', icon: { name: 'person', label: { long: 'Long label' } } }
-    ],
+    ['icon label is not a string', { name: 'Alice', icon: { name: 'person', label: null } }],
     ['icon label is empty', { name: 'Alice', icon: { name: 'Alice', label: '' } }]
-  ])('throws an error if %s', (def, prop: RecipientType) => {
-    expect(() => shallowMount(Recipient, { props: { recipient: prop } })).toThrow(
+  ])('throws an error if %s', (def: string, prop: Partial<RecipientType>) => {
+    // need to cast any here because we want to test the behaviour when "name" is empty
```

Comment:
Not sure if it's worth testing something that intentionally works around typechecks 🤷🏻‍♂️